FAQ's.md

```diff
@@ -0,0 +1,30 @@
+**Q: What is ADSdroid ?**
+
+A: It is the unofficial AllDataSheet Android Application. It gives her users the privilege of searching and downloading electronics tools in pdf format, for easier and faster access 
+
+**Q: Does it requires internet access ?**
+
+A: Absolutely. It requires internet connection, for searching and also for downloading files in PDF format.
+
+**Q: How do I use it ?**
+
+A: Very simple. Just input a valid electrical terms in the search box and click on the search button 
+
+**Q: What is the drop down-button for ?**
+
+A: When you click on the drop-down button, you'll see some options(included, start with, end with and match), these are used to make your search results more precise 
+
+**Q: How do I get my searched result as PDF ?**
+
+A: Once you've located a related search, just click on the particular one, it should download on your device and be saved as PDF in your local directory (Downloads folder) 
```

Comment:
`located a related search, just click on the particular one` should be `found the component you were looking for, just tap on it`